vtex/faststore
# Feat: `Checkbox` component.

Depends on https://github.com/vtex/faststore/pull/1496

## What's the purpose of this pull request?

~~- [x] Configure Typescript SASS and TS loaders.~~
~~- [x] Brings custom storybook components to  @faststore/ui.~~
- [x] Move Checkbox Component from @faststore/ui to @faststore/components.
- [x] Move Checkbox Styles ~~and stories~~ from nextjs.store to @faststore/ui.
- [x] Adapts docusaurus checkbox documentation.

**Question: Should we move the Checkbox props (e.g. partial) to `@faststore/ui`?** Yes
- [x] bring props from nextjs.store to @faststore/ui



## How to test it?

~~run `yarn storybook` in `package/ui` folder and verify the `Checkbox` component.~~
run `yarn docusaurus` in app/docs folderand checks the `Checkbox` component or test it using the doc preview link.

### Starters Deploy Preview





## References
